suit_generator/suit_kms_base.py

```diff
@@ -33,3 +33,20 @@ def encrypt(self, plaintext, key_name, context, aad) -> tuple[bytes, bytes, byte
         :rtype: tuple[bytes, bytes, bytes]
         """
         pass
+
+    @abstractmethod
+    def sign(self, data, key_name, algorithm, context) -> bytes:
+        """
+        Sign the data with a private key.
+
+        :param data: The data to be signed.
+        :param key_name: The name of the private key to be used.
+        :param algorithm: The name of the algorithm to be used.
+                          For file based KMS, this can be used to verify if the key in the
+                          provided file contains a key of the correct type.
+        :param context: The context to be used
```

Comment:
nit: add a period at the end of the `:param context:` description for consistency.

Reply:
Fixed